scipy/stats/_qmc.py

```diff
@@ -379,7 +420,7 @@ def primes_from_2_to(n):
     return np.r_[2, 3, ((3 * np.nonzero(sieve)[0][1:] + 1) | 1)]
 
 
-def n_primes(n):
+def n_primes(n: int) -> List[int]:
```

Comment:
This one is called by `Halton`, hence can have something else than plain `int`
```suggestion
def n_primes(n: IntNumber) -> List[int]:
```